README.md

```diff
@@ -22,6 +22,15 @@ Run the frontend portal with `npm run serve:portal` and view at http://localhost
 
 The default configuration uses remote addresses for Firebase functions. Run `npm run serve:dev:portal` to develop against locally served functions.
 
+## React Portal
+
+Run the frontend portal with `npm run serve:portal2` and view at http://localhost:4200/.
+
+<<<<<<< Updated upstream
+=======
+The default configuration uses remote addresses for Firebase functions. Run `npm run serve:dev:portal2` to develop against locally served functions.
+
+>>>>>>> Stashed changes
```

Comment:
It looks like some artifacts of a merge are in here. Seems like we can just remove them
```suggestion
The default configuration uses remote addresses for Firebase functions. Run `npm run serve:dev:portal2` to develop against locally served functions.

```